### PR TITLE
README: Add PR labeling strategy and basic development workflow

### PR DESCRIPTION
**Pull request by pdcp1**
This PR adds the PR labeling strategy to the README file.
@smcgregor and @cesarvarela if you are OK with the new `PR: Draft` PR label, please add it to the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,20 @@ The steps for contributing changes are the following,
...
+On a daily basis, all PRs should be labeled with one of the review process tags:
+- `PR: Draft`
+- `PR: Needs Review`
+- `PR: Work Needed`
+- `PR: Mergeable`
+
+The person opening the PR should apply `PR: Draft` until the work is finished, then the person can apply the `PR: Needs Review` label as soon the PR is ready to be reviewed.
+The reviewer can change it to `PR: Mergeable` or `PR: Work Needed` if the PR needs more work and assign it back to the corresponding developer.
+
+Please respond to review requests in a timely manner, or indicate another reviewer would be better. Code review is everyone’s responsibility.
```
Comment (Janetbananet) [inline README.md:43]: Please respond to review requests in a timely manner, or indicate if another reviewer would be better. Code review is everyone’s responsibility.

File: README.md [hunk 2]
```diff
@@ -29,6 +29,20 @@ The steps for contributing changes are the following,
...
 
+## PR labeling strategy
+
+On a daily basis, all PRs should be labeled with one of the review process tags:
+- `PR: Draft`
+- `PR: Needs Review`
+- `PR: Work Needed`
+- `PR: Mergeable`
+
+The person opening the PR should apply `PR: Draft` until the work is finished, then the person can apply the `PR: Needs Review` label as soon the PR is ready to be reviewed.
```
Comment (Janetbananet) [inline README.md:40]: Is it up to the person drafting the PR to assign a reviewer at the point he/she adds the label 'PR: Needs Review'?

**Comment by cesarvarela** [comment]
You can create draft pull requests, no need for another label.

**Review by cesarvarela — COMMENTED** [review]
I would rephrase the instructions to use Github's bulit-in draft pull requests. We would lose the functionality built around them if we used a label.

https://github.blog/2019-02-14-introducing-draft-pull-requests/

File: README.md [hunk 3]
```diff
@@ -29,6 +29,20 @@ The steps for contributing changes are the following,
...
+The reviewer can change it to `PR: Mergeable` or `PR: Work Needed` if the PR needs more work and assign it back to the corresponding developer.
+
+Please respond to review requests in a timely manner, or indicate another reviewer would be better. Code review is everyone’s responsibility.
+When something is mergeable, then someone else (not the implementer or reviewer) can merge it to staging. They can optionally do a final review. After merge to staging, the code quality is everyone’s responsibility.
```
[the remark this answers is not in the document]
Comment (pdcp1) [reply README.md]: I think the idea is to monitor the PRs with the "PR: Mergeable" label in a daily basis and merge it to staging.

File: README.md (hunk 2 above)
Comment (pdcp1) [reply README.md]: @Janetbananet I rephrased this sentence to be more clear about responsibilities. In my opinion, the person that works on the PR can assign someone else to review. If she/he don't know who to assign the PR to, she/he should let the team know on the Slack channel and we can follow up from there.
